Wikipedia/Code/WMFImageTagParser.h

```diff
@@ -8,6 +8,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (WMFImageTagList*)imageTagListFromParsingHTMLString:(NSString*)HTMLString;
 
+- (WMFImageTagList*)imageTagListFromParsingHTMLString:(NSString*)HTMLString withLeadImageURL:(nullable NSURL *)leadImageURL;
```

Comment:
@montehurd wouldn't that cause the image to appear twice - leadImageURL is picking an image URL out of the list to be the first